pkg/kwok/cmd/root.go

```diff
@@ -84,7 +84,7 @@ func NewCommand(ctx context.Context) *cobra.Command {
 	cmd.Flags().StringVar(&flags.Options.DisregardStatusWithAnnotationSelector, "disregard-status-with-annotation-selector", flags.Options.DisregardStatusWithAnnotationSelector, "All node/pod status excluding the ones that match the annotation selector will be watched and managed.")
 	cmd.Flags().StringVar(&flags.Options.DisregardStatusWithLabelSelector, "disregard-status-with-label-selector", flags.Options.DisregardStatusWithLabelSelector, "All node/pod status excluding the ones that match the label selector will be watched and managed.")
 	cmd.Flags().StringVar(&flags.Kubeconfig, "kubeconfig", flags.Kubeconfig, "Path to the kubeconfig file to use")
-	cmd.Flags().StringVar(&flags.Master, "master", flags.Master, "Server is the address of the kubernetes cluster")
+	cmd.Flags().StringVar(&flags.MasterUrl, "master-url", flags.MasterUrl, "MasterUrl is the address of the kubernetes cluster (https://hostname:port).")
```

Comment:
This flag is the one that comes with the kube cli and should not be changed

Reply:
ok,but the description `Server is the address of the kubernetes cluster` looks like a parameter named `Server`.
And I find the same description field in k8s named `Server`

Reply:
Sync to `kube-controller-manager`

https://kubernetes.io/docs/reference/command-line-tools-reference/kube-controller-manager/

```
--master string
The address of the Kubernetes API server (overrides any value in kubeconfig).
```

Reply:
Okay, I have edited and pushed the new code.